### DISCUSSION TITLE
Norte geográfico en la versión QGIS 3.20.0-Odense

### DISCUSSION THREAD

**Discussion by lefcgis** [qgispe/discusiones#22] [Preguntas&Respuestas] [answered]
¡Hola!

Estoy reportando un inconveniente en el cambio de simbología del norte geográfico en el compositor de mapas. (utilizo W10). Sucede que cuando utilizo la función de Añadir Flecha del Norte, por defecto me sale el símbolo nativo (fig. 1). 
![1](https://user-images.githubusercontent.com/13573242/126809177-f530c329-5253-42c8-8ea2-6ea3e7b00abf.png)

Luego, cuando quiero cambiarlo a otro símbolo, en esta versión no lo reconoce. En anteriores versiones, bastaba solo seleccionar otro símbolo y automáticamente lo reconocía. 

![2](https://user-images.githubusercontent.com/13573242/126809300-314e60d7-ca1f-4eac-abd4-222f5f3ab091.png)

¿a alguien más le está sucediendo lo mismo?. Si fue así, ¿cómo lo resolvieron sin necesariamente utilizar otros símbolos en SVG alternativos?. Mi idea es utilizar los nativos (aunque no sean tan "bonitos"). 

Gracias 😁

L.

**Comment by ambarja** [comment, marked as answer]
Hola @lefcgis  aquí alguna referencia :3 :point_down: 
https://github.com/qgis/QGIS/issues/44293

**Reply by lefcgis** [reply]
Muchas gracias @ambarja 🤓🗺️🦮
